tools/generators/epic-generator/index.ts

```diff
@@ -0,0 +1,186 @@
+import { getProjects, stripIndents, Tree } from '@nrwl/devkit';
+import { access, readFileSync } from 'fs';
+import { execSync } from 'child_process';
+import fetch from 'node-fetch';
+import { EpicGeneratorSchema } from './schema';
+
+function validateSchema(schema: EpicGeneratorSchema) {
+  if (!schema.repository.match(/[A-z-]+\/[A-z-]+/)) {
+    throw new Error('Must provide a valid repository for the issue');
+  }
+
+  if ((schema.title || '').trim() === '') {
+    throw new Error('Must provide a valid title for the issue');
+  }
+
+  return {
+    repository: schema.repository,
+    title: schema.title,
+    message: schema.message || '*Description to be added*',
+  };
+}
+
+const checkAuthentication = async () => {
```

Comment:
```suggestion
const checkAuthentication = () => {
```

seems that this function is not actually async ?